### PR TITLE
Adding how to use and sort based on recommendation

### PR DESCRIPTION
**Pull request by matsjohans**

### CODE REVIEW

File: vending-machines/README.md [hunk 1]
```diff
@@ -32,24 +32,35 @@ The QR code is a dynamic representation of the payment url, and user needs to sc
...
 
 ### When to use
 
 * The merchant redirect QR code can be used when it is not possible to present the dynamic one-time payment QR described in scenario 1. 
 
-![qr_direct_to_payment](images/1_qr_direct_to_payment.png)
+### How to use
+
+* Generate a static QR code with [merchant redirect QR](https://vippsas.github.io/vipps-developer-docs/docs/APIs/qr-api/vipps-qr-api#merchant-redirect-qr-codes) linking to a webshop connected to the specific vending machine. User select products and click the pay button that generate a [CreatePaymentRequest](https://vippsas.github.io/vipps-developer-docs/docs/APIs/epayment-api/getting-started#step-2---create-a-payment) based on the selected products.
+Specify `"customerInteraction": "CUSTOMER_PRESENT"` and `"userFlow": "WEB_REDIRECT"` to redirect user to Vipps.
```
Comment (cloveras) [inline vending-machines/README.md:60]:
```suggestion

Specify `"customerInteraction": "CUSTOMER_PRESENT"` and `"userFlow": "WEB_REDIRECT"` to redirect user to Vipps. 
```

File: vending-machines/README.md [hunk 2]
```diff
@@ -58,15 +69,34 @@ A merchant-generated QR code is posted on the vending machine.
 When the customer scans the QR code, they are taken to the merchant's landing page, which is waiting for the product to be selected on the vending machine.
 The price is presented and the user pays for the product in their Vipps app.
 
+### How to use
+
+* Generate a static QR code with [merchant redirect QR](https://vippsas.github.io/vipps-developer-docs/docs/APIs/qr-api/vipps-qr-api#merchant-redirect-qr-codes) linking to a web page containing the selected product from the vending machine. User clicks the pay button that generate a [CreatePaymentRequest](https://vippsas.github.io/vipps-developer-docs/docs/APIs/epayment-api/getting-started#step-2---create-a-payment) with the selected amount.
+Specify `"customerInteraction": "CUSTOMER_PRESENT"` and `"userFlow": "WEB_REDIRECT"` to redirect user to Vipps.
```
Comment (cloveras) [inline vending-machines/README.md:75]: Add a blank line?
```suggestion

Specify `"customerInteraction": "CUSTOMER_PRESENT"` and `"userFlow": "WEB_REDIRECT"` to redirect user to Vipps. 
```

File: vending-machines/README.md [hunk 3]
```diff
@@ -58,15 +69,34 @@ A merchant-generated QR code is posted on the vending machine.
...
+The payment amount should be the max amount of the vending machine products. After reservation, the amount of the selected product must be captured, and the remaining amount must be released. 
+
+### When to use
+
+* The merchant redirect QR code can be used when it is not possible to present the dynamic one-time payment QR described in scenario 1. 
+
+### How to use
+
+* Generate a static QR code with [merchant redirect QR](https://vippsas.github.io/vipps-developer-docs/docs/APIs/qr-api/vipps-qr-api#merchant-redirect-qr-codes) linking to a web service that generates a [CreatePaymentRequest](https://vippsas.github.io/vipps-developer-docs/docs/APIs/epayment-api/getting-started#step-2---create-a-payment) with a specific amount.
+Specify `"customerInteraction": "CUSTOMER_PRESENT"` and `"userFlow": "WEB_REDIRECT"` to redirect user to Vipps.
```
Comment (cloveras) [inline vending-machines/README.md:96]: Add a blank line?```suggestion

Specify `"customerInteraction": "CUSTOMER_PRESENT"` and `"userFlow": "WEB_REDIRECT"` to redirect user to Vipps. 
```

File: vending-machines/README.md [hunk 4]
```diff
@@ -58,15 +69,34 @@ A merchant-generated QR code is posted on the vending machine.
...
 
-When the customer scans the QR code,
-they are taken to the merchant's landing page, where products can selected.
-The price is presented and the user pays for the product in their Vipps app.
+A [Vipps QR code](https://vippsas.github.io/vipps-developer-docs/docs/APIs/qr-api/vipps-qr-api#merchant-redirect-qr-codes) is posted on the vending machine.
 
-![3_qr_to_landing_page_providing_selection](images/3_qr_to_landing_page_providing_selection.png)
+When the customer scans the QR code, they go directly to the Vipps payment screen on their phone, where they can approve the payment.
+
+The payment amount should be the max amount of the vending machine products. After reservation, the amount of the selected product must be captured, and the remaining amount must be released.
```
Comment (cloveras) [inline vending-machines/README.md:87]: Link to the cancel endpoint?

**Comment by rebekaburnett** [comment]
That looks fantastic!!

**Review by rebekaburnett — APPROVED** [review]

**Pull request merged**